10_K_Risk_Evaluator/html_parser.py

```diff
@@ -43,6 +43,7 @@ def grab_section_text(soup_object, start_terms, end_terms):
                 # Located the start of 'Item 1A: Risk Factors'
                 within_item_1A = True
                 section_text.append(tag)
+                import pdb;pdb.set_trace()
```

Comment:
Should we move all import calls to the top of the script?
What does pdb.set_trace() do?

Reply:
import pdb;pdb.set_trace(). is a debugging tool, it's not truly like a import package. It stops the code at that line and i can see what each variable is up into that point. I had it there to understand that function but it can be removed in the final copy.